projects/arcade-xyz/index.js

```diff
@@ -11,12 +12,38 @@ const {
   VAULT_FACTORY_A,
 } = require('./constants');
 
+// to run: node test.js projects/arcade-xyz/index.js
+
 // Uses chainlink oracle floor price for all whitelisted NFTS owned by every vault and the Loan Core contract.
 // Tokens owned by vaults have been wrapped into an Arcade.xyz vault. Tokens owned by the Loan Core contract
 // are currently in escrow.
+
+async function getAvailableBlock(api, decrement , attempts) {
```

Comment:
This is a helpful idea for testing but in practice it will produce the wrong information since the data returned by our adapter will be tied to a specific block by DefiLlama. In production the adapter is run with some delay so this never happens. I would just hardcode the block number while testing.

It's too bad we can't use `api.getBlock` as suggested by the repo maintainers. You can stub it for testing and read from an env_var if you really don't want to hardcode it.